db/multi_db.go

```diff
@@ -0,0 +1,99 @@
+package db
+
+import (
+	"github.com/rakutentech/go-echo-kit/config"
+	"github.com/rakutentech/go-echo-kit/logger"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+	gormlogger "gorm.io/gorm/logger"
+	"gorm.io/plugin/dbresolver"
+
+	"log"
+	"os"
+	"sync"
+	"time"
+)
+
+var onceMultiDb sync.Once
+var SingletonDB *gorm.DB
+
+type multiDbConf struct {
+	Master string // master db dsn
+	Slave  []string // slave db dsn
```

Comment:
As it takes multiple ``Slaves`` is better. 
Would you be able to add the usage upon multiple slaves inside README.md